examples/multiplayer-netfox/scripts/player.gd

```diff
@@ -16,7 +16,7 @@ func _ready():
 	await get_tree().process_frame
 	$RollbackSynchronizer.process_settings()
 
-func _tick(delta: float, _tick: int):
```

Comment:
I just noticed that `_tick` had type hints, but the `_rollback_tick` line I copied from `res://examples/forest-brawl/scripts/brawler-controller.gd` does not. Is there a convention you follow for this sort of thing? If not, then I think it's good to go.

Reply:
That's purely my inconsistency :slightly_smiling_face: I thinkthis will be good as-is.